content/03-reference/02-database-connectors/04-mysql.mdx

```diff
@@ -195,3 +195,7 @@ model Device {
 //  data   xml
 }
 ```
+
+## Engine
+
+If you are using a version of MySQL where InnoDB is not the default engine, you must specify `ENGINE = InnoDB;` when you create a table. If you introspect a database that uses a different engine, relations in the Prisma Schema are not created (or lost, if the relation already existed).
```

Comment:
Is that correct that only InnoDB tables are introspected to have relations? What are possible other engines?

Reply:
@janpio It seems there are many (Blackhole, CSV). MyISAM is specifically the one that doesn't work. I can change it to "If you are using a version of MySQL that uses MyISAM, specify `Engine=InnoDB;` when you create a table". That way we aren't explicitly saying that no other engine works.

Reply:
I've changed it to what I suggested.